install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -764,6 +764,8 @@ encryption:
     # -- Maximum duration of the IPsec key rotation. The previous key will be
     # removed after that delay.
     keyRotationDuration: "5m"
+    # -- Enable IPSec encrypted overlay
+    encryptedOverlay: true
   wireguard:
```

Comment:
Hmm, why do we default to `true` here? I would tend towards being conservative, and starting with an explicit opt-in.

Reply:
hmm, 

You are right, but I'm not sure when this tmpl is used, our defaults inside Cilium are

```
	// Enable IPSec encrypted overlay
	//
	// This feature will encrypt overlay traffic before it leaves the cluster.
	EnableIPSecEncryptedOverlay = false
```

Anyway, it should be false but now I'm a bit curious.

Reply:
fixed in latest push, resolving.